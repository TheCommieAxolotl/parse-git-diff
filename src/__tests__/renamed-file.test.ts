import { getFixture } from './test-utils';
import parseGitDiff from '../parse-git-diff';

describe('renamed-file', () => {
  const fixture = getFixture('renamed-file');

  it('parse `renamed-file`', () => {
    expect(parseGitDiff(fixture)).toMatchSnapshot();
  });
});
