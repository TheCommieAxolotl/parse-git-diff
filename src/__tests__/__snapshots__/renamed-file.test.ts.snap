// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renamed-file parse \`renamed-file\` 1`] = `
{
  "files": [
    {
      "chunks": [],
      "pathAfter": "rename.md",
      "pathBefore": "newfile.md",
      "type": "RenamedFile",
    },
  ],
  "type": "GitDiff",
}
`;
